### PR TITLE
Nx Go Support

### PR DESCRIPTION
**Pull request by kinnairdclan**
Nx Go Dockerfile generation support.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,13 +4,13 @@ This is a project to build and publish docker images for various [Network Optix]
 
 ## License
 
-Licensed under the [MIT License][license].  
+Licensed under the [MIT License][license].
```
Comment (ptr727) [inline README.md:7]: Please revert the intentional double space at end of lines for newline behaviors.  
Probably related to the remove spaces at end of lines on save option, should be disabled for e.g. markdown and text files.

E.g. workspace settings:

```json
		"files.trimTrailingWhitespace": true,
		"[markdown]": {
			"files.trimTrailingWhitespace": false
		},
		"[plaintext]": {
			"files.trimTrailingWhitespace": false
		},
		"diffEditor.ignoreTrimWhitespace": false,
		"editor.renderWhitespace": "boundary"
```

**Review by ptr727 — CHANGES_REQUESTED** [review]
Thank you for the PR, please address the requested changes.

File: README.md (hunk 1 above)
Comment (kinnairdclan) [reply README.md]: My pleasure, very cool and organized project you have here.
Reverted README changes, operated on README after opening the proper workspace in VSCode and the trailing whitespace was not removed (see latest diff).

**Comment by ptr727** [comment]
Please make the required changes to e.g. [https://github.com/ptr727/NxWitness/blob/main/Make/Instructions.sh] echo'ing the new port numbers assigned in the compose stack.

**Comment by kinnairdclan** [comment]
Please see updates. Also targeted README Docker Hub information.

**Comment by ptr727** [comment]
All builds passed, I merged the change, if you find any issues feel free to PR again, please PR against `develop` branch, I should add a CONTRIBUTING guide.

**Pull request merged**